sdk/nodejs/appservice/zMixins.ts

```diff
@@ -400,6 +400,46 @@ export interface Function {
     appSettings?: pulumi.Input<{ [key: string]: any }>;
 }
 
+// Type guard for Function interface

```

Comment:
FWIW, i think this comment is unnecessary :)

Reply:
It's the first time I write a guard, so I thought it's a big thing :) will remove